prime-router/src/main/kotlin/Metadata.kt

```diff
@@ -193,6 +193,9 @@ class Metadata {
         }
         val fullElement = if (baseElement != null) element.inheritFrom(baseElement) else element
 
+        if (fullElement.maxLength != null && fullElement.maxLength < 0)
+            error("Schema Error: maxLength ${fullElement.maxLength} for ${fullElement.name} must be >= 0")
```

Comment:
Can the maxlength be zero? Should it ever be zero? Should we have a max max length?

Reply:
Yes, zero is allowed - I saw no reason to disallow it.    Since we control this internally, I don't see a need for a max max length, at least not at the moment.